integration-tests-jvm/azure-servicebus/pom.xml

```diff
@@ -58,6 +58,22 @@
             <groupId>io.quarkus</groupId>
             <artifactId>quarkus-resteasy</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.apache.camel.quarkus</groupId>
+            <artifactId>camel-quarkus-direct</artifactId>
```

Comment:
Whenever you add new Camel Quarkus extension dependencies to the test, the corresponding `-deployment` dependency needs to be added to the `virtualDependencies` profile (you'll find it at the end of the `pom.xml`).